alignak/util.py

```diff
@@ -1308,6 +1308,10 @@ def parse_daemon_args(arbiter=False):
                                  'multiple -c can be used, they will be concatenated')
         parser.add_argument("-V", "--verify-config", dest="verify_only", action="store_true",
                             help="Verify config file and exit")
+        parser.add_argument("-n", "--config-name", dest="config_name",
```

Comment:
We pass the arbiter name when start, will be useful for spare feature (when we will code it).
The second problem with the spare it's each arbiter (master and spare) try get name with is_me() but can't really use it because not get really arbiter name.